File: 3st/Juram/programmer/대문자와 소문자.js
function solution(my_string) {
    var answer = '';
    for(let i=0; i<my_string.length; i++){  
        if(my_string[i] === my_string[i].toUpperCase()) {
            answer += my_string[i].toLowerCase()
        }else{
            answer += my_string[i].toUpperCase()
        }
    }
    return answer;
}

File: 5st/gyuri/대문자와 소문자.java
class Solution {
    public String solution(String my_string) {
        String answer = "";
        
        for(char c : my_string.toCharArray()){
        if(Character.isUpperCase(c)){
            answer = answer + Character.toLowerCase(c);
        }else{
            answer = answer + Character.toUpperCase(c);
        }
        }
        return answer;
    }
}